### ISSUE TITLE
Make bids_corrections.py report all changes made to one CHANGELOG

### ISSUE THREAD

**Issue by ericearl** [nimh-dsst/abcd-fasttrack2bids#16]
This will log all the adjustments made from the original data as converted by Dcm2Bids/dcm2niix to the data as it is converted after the bids_corrections.py stage.

**Comment by ericearl** [comment]
I'm thinking now it should be a TSV file with a data dictionary something like:

```json
{
    "time": "time of correction",
    "option": "option that triggered correction",
    "file": "array of file(s) which were corrected",
    "field": "field undergoing correction",
    "original_value": "original pre-corrected value",
    "corrected_value": "corrected value"
}
```

**Comment by ericearl** [comment]
Solved with commit 7e9bd71
